docker/dockerfile-r

```diff
@@ -4,7 +4,9 @@ FROM rocker/verse:${R_VERSION}
 WORKDIR /lgbm
 
 RUN apt-get update && \
-    apt-get install -y build-essential cmake && \
+    apt-get install -y --no-install-recommends \
+        build-essential \
+        cmake && \
```

Comment:
I support this change, but it seems unrelated. Could you put it in a separate PR that we could approve and merge more quickly (since this PR will have to wait for a while until we know if the next release will be 3.1.1 or 3.2.0)?

Reply:
OK, sure!